Documentation/joss/notes.md

```diff
@@ -17,7 +17,7 @@ Yes, GPLv3.0 - see [licence](../../Licence.txt).
 
 > * The software should have an obvious research application.
 
-See list of publications made with OIP (TUM Open Infra Platform) [here](./paper.bib).
+See list of publications made with OIP (TUM OpenInfraPlatform) [here](./paper.bib).
```

Comment:
Was named in the past always "TUM Open Infra Platform" with whitespaces - all the Titel of paper.md  contains the name of OIP with whitespace - at least make it unique across the whole repository/paper

Reply:
Yeah, makes sense. Will change everywhere accordingly.

Reply:
resolved in 9060ca9b6f38a8d0a64a4bed34b6c34e4f1dd983